### DISCUSSION TITLE
モバイル学習コンテンツで、文言の削除漏れか要確認

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#262] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/screens/todo-board
ToDo一覧画面

REST API呼び出しのセクションがすでにあるため、以下の文言は削除漏れなのか要確認。

> 現在のバージョンではバックエンド連携は実装していません。

**Comment by nagano-ryo** [comment]
pull request
https://github.com/ws-4020/mobile-app-crib-notes/pull/273

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
